itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcidTablesBase.java

```diff
@@ -203,18 +203,17 @@ void verifyLoadExecution(String replicatedDbName, String lastReplId, boolean inc
 
   void prepareIncAcidData(String dbName) throws Throwable {
     primary.run("use " + dbName)
-            .run("create table t6 (str string) stored as orc tblproperties " +
-                    "(\"transactional\"=\"true\")")
-            .run("insert into t6 values ('aaa'), ('bbb')")
+            .run("create table t6 stored as orc tblproperties (\"transactional\"=\"true\")" +
+                    " as select * from t1")
             .run("alter table t2 add columns (placetype string)")
             .run("update t2 set placetype = 'city'");
     acidTableNames.add("t6");
   }
 
   private void verifyIncAcidLoad(String dbName) throws Throwable {
     replica.run("use " + dbName)
-            .run("select str from t6 order by str")
-            .verifyResults(new String[]{"aaa", "bbb"})
+            .run("select id from t6 order by id")
+            .verifyResults(new String[]{"1", "2"})
```

Comment:
change insertRecordsIntoDB method in ReplicationTestUtils to take table property and table storage for ctas

Reply:
When I corrected the testcase as per your suggestion, I came across two bugs: https://issues.apache.org/jira/browse/HIVE-21678 and https://issues.apache.org/jira/browse/HIVE-21679. Looks like we don't have enough coverage for CTAS. So, I have created a parent JIRA https://issues.apache.org/jira/browse/HIVE-21677 for changing this effectively. For now, I think, it will be better go ahead without changing those testsets and commit just this fix.